Vagrantfile

```diff
@@ -1,4 +1,11 @@
 targets = {
+  "debian-wheezy" => {
```

Comment:
minor nit, for consistency in vagrant targets, let's stick to version number instead of code-names.